lib/Runtime/Debug/DebugContext.cpp

```diff
@@ -49,6 +60,11 @@ namespace Js
         }
     }
 
+    bool DebugContext::IsSelfOrScriptContextClosed() const
+    {
+        return (this->IsClosed() || this->scriptContext->IsClosed());
```

Comment:
No, scriptContext on debugContext is never set to null (in constructor we have assert that initial value is not null)